### PR TITLE
Rename TrackPaneData to TimerData

### PR DESCRIPTION
**Pull request by karupayun**
This boring PR is only a renaming. This was discussed in
https://github.com/google/orbit/pull/2760#discussion_r707781487.

### CODE REVIEW

File: src/ClientData/include/ClientData/TimerData.h [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 namespace orbit_client_data {
 
-class TrackPaneData final {
+class TimerData final {
```
Comment (dimitry-) [inline src/ClientData/include/ClientData/TimerData.h:16]: this class has a number of timers not just one. I think the name is misleading... maybe TimerDataManager?

Comment (karupayun) [reply src/ClientData/include/ClientData/TimerData.h:16]: Is the data related to the Timers or TimerData. Could be TimersData, but I'm happy with TimerData. We already have a TimerDataManager (creates TimerData) and in the future we will have a TimerDataProvider instead, as well as I am creating a ScopeTreeTimerDataProvider (PR will come soon).

Comment (dimitry-) [reply src/ClientData/include/ClientData/TimerData.h:16]: I would prefer something more descriptive maybe TrackDataCollection?

Comment (karupayun) [reply src/ClientData/include/ClientData/TimerData.h:16]: I already named ScopeTreeTimerData, I should rename that in the same way. What do you think @pierricgimmig ?

Comment (pierricgimmig) [reply src/ClientData/include/ClientData/TimerData.h:16]: We already have CallstackData and TracePointData which both represent data on a number of elements. I think TimerData fits well with the current nomenclature.

**Review by pierricgimmig — APPROVED** [review]

**Pull request merged**